util/fipstools/acvp/modulewrapper/modulewrapper.cc

```diff
@@ -2058,6 +2102,36 @@ static bool FFDH(const Span<const uint8_t> args[], ReplyCallback write_reply) {
   return write_reply({BIGNUMBytes(DH_get0_pub_key(dh.get())), z});
 }
 
+static bool PBKDF(const Span<const uint8_t> args[], ReplyCallback write_reply) {
+  const Span<const uint8_t> password = args[0];
+  const Span<const uint8_t> salt = args[1];
+  const Span<const uint8_t> iterations = args[2];
+  const Span<const uint8_t> hmac_name = args[3];
+  const Span<const uint8_t> key_len = args[4];
+
+  // read bit data into useable variables
+  unsigned int iterations_uint;
+  memcpy(&iterations_uint, iterations.data(), sizeof(iterations_uint));
+  unsigned int key_len_uint;
+  memcpy(&key_len_uint, key_len.data(), sizeof(key_len_uint));
+
+  key_len_uint = key_len_uint/8;
```

Comment:
Any restrictions in key length we should catch here?

Reply:
Restrictions in key length are caught in [line 99](https://github.com/awslabs/aws-lc/pull/657/files#diff-5ad898a0522db58a0c3e9e6084109fde76013892781d282bc86b86e17afc3a52R99) of `pbkdf.go`, which is where key_len is obtained from - so I don't think its necessary here.